ambari-server/src/test/java/org/apache/ambari/server/agent/TestHeartbeatHandler.java

```diff
@@ -1317,25 +1317,14 @@ public void testComponents() throws Exception {
     expected.setComponents(dummyComponents);
 
     Cluster cluster = heartbeatTestHelper.getDummyCluster();
-    Service service = EasyMock.createNiceMock(Service.class);
-    expect(service.getName()).andReturn("HDFS").atLeastOnce();
-
+    Service service = cluster.getService("HDFS");
     Map<String, ServiceComponent> componentMap = new HashMap<>();
     ServiceComponent nnComponent = EasyMock.createNiceMock(ServiceComponent.class);
-    expect(nnComponent.getName()).andReturn("NAMENODE").atLeastOnce();
-    expect(nnComponent.getStackId()).andReturn(dummyStackId).atLeastOnce();
+    expect(nnComponent.getName()).andReturn("NAMENODE").anyTimes();
+    expect(nnComponent.getStackId()).andReturn(service.getStackId()).anyTimes();
+    replay(nnComponent);
     componentMap.put("NAMENODE", nnComponent);
-
-    expect(service.getServiceComponents()).andReturn(componentMap).atLeastOnce();
-    expect(service.getServiceId()).andReturn(1L).atLeastOnce();
-    expect(service.getServiceType()).andReturn("HDFS").atLeastOnce();
-    expect(service.getStackId()).andReturn(dummyStackId).atLeastOnce();
-
-    ActionManager am = actionManagerTestHelper.getMockActionManager();
-
-    replay(service, nnComponent, am);
-
-    cluster.addService(service);
+    service.addServiceComponent(nnComponent);
 
     HeartBeatHandler handler = heartbeatTestHelper.getHeartBeatHandler(am);
```

Comment:
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/jenkins/jenkins-slave/workspace/Ambari-Github-PullRequest-Builder/ambari-server/src/test/java/org/apache/ambari/server/agent/TestHeartbeatHandler.java:[1329,72] cannot find symbol
  symbol:   variable am
  location: class org.apache.ambari.server.agent.TestHeartbeatHandler
```

Reply:
ok, I knew it. When I resolved conflicts online, I accidentally removed 
ActionManager am = actionManagerTestHelper.getMockActionManager();
I will resubmit it.